### DISCUSSION TITLE
Change NOR -> SEL while a modifier is pressed

### DISCUSSION THREAD

**Discussion by dstrelnikov** [helix-editor/helix#10192] [General]
Would that be possible to assign a modifier (for instance `Alt`) to switch the normal mode (NOR) to select / extend mode SEL mode while the modifier is being hold?

Something like `v2wv`  = `Mod(hold) 2w Mod(release)`.

**Comment by archseer** [comment]
This is impossible to do in the terminal because we only get press events (+ repeated events on hold), no release/hold events.
